core/src/main/java/org/axonframework/commandhandling/distributed/ConsistentHashChangeListener.java

```diff
@@ -25,6 +25,8 @@
 @FunctionalInterface
 public interface ConsistentHashChangeListener {
 
+    ConsistentHashChangeListener NO_OP_CONSISTENT_HASH_CHANGE_LISTENER = newConsistentHash -> { };
```

Comment:
This line can be removed. We've replaced the usage of this constant for the `noOp()` function below.